index.Rmd

```diff
@@ -131,58 +124,57 @@ In its most basic use-case, `read_csv` expects that the data file:
 
-Next, we call the `read_csv` function and pass it a single argument: the name of the file, `"can_lang.csv"`. We have to put quotes around filenames and other letters and words that we 
-use in our code to distinguish it from the special words that make up R programming language.  This is the only argument we need to provide for this file, because our file satifies everthing else 
-the `read_csv` function expects in the default use-case (which we just discussed). Later in the course, we'll learn more about how to deal with more complicated files where the default arguments are not 
-appropriate. For example, files that use spaces or tabs to separate the columns, or with no column names.
+Below you'll see the code used to load the data into R using the `read_csv` 
+function. However, given that `read_csv` is not included in the base 
+installation of R, we will have to load it from somewhere else before we can use
+it. The place we will load it from is an R package. An R package is a collection
+of functions that once loaded, can be used in addition to the built-in R package
+functions. The `read_csv` function, in particular, can be made accessible 
+through loading the `tidyverse` package (more on this later), using the 
+`library` function.
+
+After loading the `tidyverse` package, we can call the `read_csv` function and 
+pass it a single argument: the name of the file, `"can_lang.csv"`. We have to put quotes around file names and other letters and words that we use in our code to distinguish it from the special words that make up the R programming language.  The file's name is the only argument we need to provide because our file satisfies everything else the `read_csv` function expects in the default use-case (which we just discussed). Later in the course, we'll learn more about dealing with more complicated files where the default arguments are not appropriate, for example, files that use spaces or tabs to separate the columns instead of commas.
 
 ```{r load_state_property_data, warning=FALSE, message=FALSE}
 library(tidyverse)
 read_csv("data/can_lang.csv")
 ```
-Above you can also see something neat that Jupyter does to help us understand our code: it colours text depending on its meaning in R. For example, 
-you'll note that functions get bold green text, while letters and words surrounded by quotations like filenames get blue text.
+
 
 > **In case you want to know more (optional):** 
-> We use the `read_csv` function from the `tidyverse` instead of the base R function `read.csv` because it's faster and it creates a nicer variant of the base R data frame called a *tibble*. 
+> We use the `read_csv` function from the `tidyverse` instead of the base R function `read.csv` because it's faster, and it creates a nicer variant of the base R data frame called a *tibble*. 
 > This has several benefits that we'll discuss in further detail later in the course.
 
-## Assigning value to a data frame
+## Giving a data frame a name
```

Comment:
Could we call this, "Naming things"? This topic is more general than just data frames. Then we could also add a couple sentences on what are good names? E.g., they should be syntactic (you don't need to use this word, but summarize what qualifies as this, as specified here: https://principles.tidyverse.org/names-attribute.html#universal-names) and recommend that we follow the tidyverse naming convention in this book (snake_case).